src/searcher.rs

```diff
@@ -196,8 +196,8 @@ impl Searcher {
 #[pyclass]
 #[derive(Clone, Debug)]
 pub(crate) struct DocAddress {
-    pub(crate) segment_ord: tv::SegmentLocalId,
-    pub(crate) doc: tv::DocId,
+    pub(crate) segment_ord: tv::SegmentOrdinal,
+    pub(crate) doc_id: tv::DocId,
```

Comment:
any reason for changing the public interface here? that has stability implications for extant users.

Reply:
Since it is pub(crate) it should only impact the crate itself (I think).  I just updated the naming to match the tantivy changes.  It doesn't matter to me either way.

Reply:
ic, thanks for clarifying and verified.